runtime/Cpp/runtime/src/misc/InterpreterDataReader.h

```diff
@@ -6,6 +6,8 @@
 #pragma once
 
 #include "antlr4-common.h"
+#include "atn/ATN.h"
```

Comment:
Why's that here now?

Reply:
This is to make the header compilable by itself, rather than relying on include order. This is necessary for some tooling.